doc/classes/Control.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Control" inherits="CanvasItem" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Base node for all User Interface components.
+		All User Interface nodes inherit from Control. A Control anchors anywhere within its parent node's bounding box. It features margins relative to the anchor's position, that update automatically.
```

Comment:
Maybe a little bit unclear. I would say instead something like "The Control node features an anchors/margins system to dynamically adapt its position an size relatively to its parent."